docs/docker.rst

```diff
@@ -1,17 +1,75 @@
 Running with Docker
 ===================
 
+The easiest way to get started is by using
+`docker-compose <https://docs.docker.com/compose/>`__.
+
+Promgen ships with two ``docker-compose.yml`` files:
```

Comment:
Since the two files were combined, I think these docs need to also be updated :)

Reply:
Yeah, true :smile:  I'll update the documentation when this PR is about to settle.